recipes/simplification/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "simplification" %}
+{% set version = "0.5.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/urschrei/{{ name }}/tarball/{{ version }}
```

Pretty sure this needs a `v` and a `.tar.gz`:
```suggestion
  url: https://github.com/urschrei/{{ name }}/tarball/v{{ version }}.tar.gz
```

---

you're right about the v
the extension is not there... i tried both paths.
thanks